tests/30-main.spec.js

```diff
@@ -700,11 +695,28 @@ describe('main', () => {
     err.message.should.contain('@context" must include');
   });
 
-  it('should fail when credentialStatus is not an object for ' +
+  it('should fail when "credentialStatus" is not an object for ' +
     '"getCredentialStatus"', async () => {
-    const id = 'https://example.com/status/1';
-    const list = await createList({length: 100000});
-    const credential = await createCredential({id, list});
+    const credential = {
```

Comment:
Why was this changed to be hard-coded vs. using the `createCredential` function?

Reply:
This was changed to be hard-coded since `createCredential()` doesn't return a credential with "credentialStatus" property on it.

Reply:
But it gets deleted below anyway (line 723) ... right? Isn't the point of this test to check for when it's not present?

Reply:
Yeah, makes sense. I've reverted it back to how it was previously. [e7c328b](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/e7c328bbaaf807a9ac9add67438aa6d8f8a06d70)